src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Lottos {
+    private final List<Lotto> lottos;
+
+    Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+
+    public int size() {
+        return lottos.size();
+    }
+
+    public List<Lotto> getLottos() {
+        return lottos;
```

Comment:
외부에서 lottos에 Lotto를 추가 및 삭제 할 수 없도록 해주세요. 

```suggestion
        return Collections.unmodifiableList(lottos);
```

Reply:
앗 넵!